update.py

```diff
@@ -43,8 +45,9 @@
 import argparse
 parser = argparse.ArgumentParser(description='Copy Common Files as needed, stripping out non-relevant wiki content')
 parser.add_argument('--site', help="If you just want to copy to one site, you can do this. Otherwise will be copied.")
-parser.add_argument('--clean', default='False', help="Does a very clean build - resets git to master head (and TBD cleans up any duplicates in the output).")
-parser.add_argument('--cached-parameter-files', default=False, help="Do not re-download parameter files", type=bool)
+parser.add_argument('--clean', action='store_true', help="Does a very clean build - resets git to master head (and TBD cleans up any duplicates in the output).")
+parser.add_argument('--cached-parameter-files', action='store_true', help="Do not re-download parameter files", default=False)
```

Comment:
Nitpicking: the default argument can be removed since that's added by the new action `store_true`.

Reply:
I know, I just like making it explicit